man/bootup.xml

```diff
@@ -225,7 +225,7 @@ emergency.service    |              |              |
     <filename>/sysroot</filename>, the
     <filename>sysroot.mount</filename> unit becomes active and
     <filename>initrd-root-fs.target</filename> is reached. The service
-    <filename>initrd-parse-etc.service</filename> scans
+    <filename>initrd-sysroot-fstab.service</filename> scans
```

Comment:
Is it possible that external units have deps on the old name? Should it have an alias for a while?